docs/architecture/adr-012-problem-plugin-architecture.html

```
@@ -0,0 +1,337 @@
+      "metaPath": "/meta",          // 各 slot で platform 自己申告を読む
+      "scorePath": "/score"
+    },
+    "platformRules": {
+      "ec2":       { "points": 100,  "degradedPoints": 10 },
+      "lambda":    { "points": 1000 },
+      "ecs":       { "points": 1000 },
+      "apprunner": { "points": 1000 }
+    },
+    "failurePenalty": -100,
+    "responsePenalties": [
+      { "if": "responseTimeMs > 1500", "points": 10 }
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)